src-browserify/api/core-lib/_promise.js

```diff
@@ -1,13 +1,16 @@
-var Backbone = window.Backbone || require('backbone-isch');
+// NOTE this does not return a _Promise directly, but a wrapper, to inject the dependencies
+// e.g. var _Promise = require('./_promise')(Backbone.Events)
+// @param {Object} BackboneEvents
```

Comment:
I'll think I finally scrap these kind of comments, they're a bit tedious to write/maintain and most likely they're going to get out-of-date like usually happens when things start to be moved around. The dependencies names' are essentially self-explanatory anyway, I think.